extensions-builtin/Lora/ui_extra_networks_lora.py

```diff
@@ -66,9 +68,9 @@ def create_item(self, name, index=None, enable_filter=True):
         return item
 
     def list_items(self):
-        for index, name in enumerate(networks.available_networks):
+        names = list(networks.available_networks)

```

Comment:
but isn't that using the same object?
the whole point was to make this thread safe

Reply:
ya
using `networks.available_networks.keys()` cause 
`RuntimeError: dictionary changed size during iteration`

I didn't want to create a new object that's why I go for the thread lock method before
but also say he prefers no lock

Reply:
> the whole point was to make this thread safe

My bad, I didn't pay enough attention :smile: 

In this case, converting into list is indeed the right approach, but then I think it would be good to add a small comment explaining created this list is actually needed to protect against concurrent modification of the dict during the iteration

Reply:
good point